app/helpers/application_helper.rb

```diff
@@ -597,4 +597,11 @@ def concurrent_events(event)
     end
     concurrent_events
   end
+
+  def speaker_selector_input(form)
+    users = User.select(:id, :name, :email).where(is_disabled: false).order(:name)
```

Comment:
This does not show the records ordered. 
We order by name, but some of the records do not have a name (so name is nil). When we do `users.map` in the collection and call `user.name` it either shows the name or the username of the user (due to https://github.com/openSUSE/osem/blob/master/app/models/user.rb#L82)

As such the first records showing up could be a username that begins with z, which doesn't make a lot of sense when you want your records to be ordered.

If you run this in rails console you will see an error caused by this.